tools/cli/air-trace/src/stats.rs

```diff
@@ -48,6 +48,7 @@ pub(crate) fn stats(mut args: Args) -> anyhow::Result<()> {
 
     let mut stats = self::report::StatsReport::new();
 
+    #[allow(clippy::significant_drop_in_scrutinee)]
```

Comment:
One can avoid this warning by passing `&mut stdin` instead (making `mut stdin` above).
I don't see it in my installation, BTW.

Reply:
Looks like the reason here is `anyhow::Result`.

Reply:
The lint seems to be broken:
https://github.com/rust-lang/rust-clippy/issues/8987